cpp/src/arrow/engine/substrait/serde.h

```diff
@@ -89,8 +89,7 @@ ARROW_ENGINE_EXPORT Result<std::vector<acero::Declaration>> DeserializePlans(
 /// \param[out] ext_set_out if non-null, the extension mapping used by the Substrait
 /// \param[in] conversion_options options to control how the conversion is to be done.
 /// Plan is returned here.
-/// \return an ExecNode corresponding to the single toplevel relation in the Substrait
-/// Plan
+/// \return an ExecPlan for the Substrait Plan
```

Comment:
Revert this?

Reply:
The pre-PR comment is incorrect, so I fixed it.